app/dashboards/hold_dashboard.rb

```diff
@@ -54,9 +58,11 @@ class HoldDashboard < Administrate::BaseDashboard
   date_requested
   date_start
   date_end
+  dates_thesis_files_received
```

Comment:
I think it would be good to include `date_released` on the show page (not sure if that was in the requirements, but it seems useful to include it on this detail page rather than just on the list pages)

Reply:
It's not in the requirements, so I'm hesitant to add it now. Would you be comfortable waiting to see if that comes up in stakeholder review?